### PR TITLE
Feature/iot_1228_multicast_docs

### PR DESCRIPTION
**Pull request by augusthjerrild**
Documentation for multicast-groups in OS2IoT.

### CODE REVIEW

File: source/multicast/multicast.rst [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+of devices instead of sending many downlink payloads to devices one by one.
+
+Multicast-groups does not support Class-A devices, and at this moment, multicast-groups only supports LoRaWAN devices.
+
+Creation in OS2IoT
+-------------------
+If you want to make a multicast-group in OS2IoT you have to go into a specific application and then create the multicast-group from there.
+
+You have to fill out all the required forms, and in the end you can add some devices. It's required that the devices share the same service profile, which is set when you create a LoRaWAN device.
+The multicast-address has to be a 8-char hex value, and the network and application session key has to be a 32-char hex value.
```
[the remark this answers is not in the document]
Comment (AramAlsabti) [reply source/multicast/multicast.rst:20]: "Hex" hasn't been mentioned in any of the other pages. So, for the first-time use, let's make it clear what the abbreviation is of like so:

"The multicast-address has to be an 8-character hexadecimal (hex) value (...)".
Now, we've established the shorthand form to the reader. We're free to use it for the remainder of this page.

**Review by AramAlsabti — CHANGES_REQUESTED** [review]

File: source/multicast/multicast.rst [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+You have to fill out all the required forms, and in the end you can add some devices. It's required that the devices share the same service profile, which is set when you create a LoRaWAN device.
+The multicast-address has to be a 8-char hex value, and the network and application session key has to be a 32-char hex value. 
+
+If these requirements are not met, then the multicast-group will not be created.
+
+When you have added the devices that have the same service profile, and have filled out the other forms, then a multicast-group will be created in the database and in Chirpstack.
+
+This sequence diagram describes the flow from the user to the Chirpstack Application Server:
+|image1|
```
[the remark this answers is not in the document]
Comment (augusthjerrild) [reply source/multicast/multicast.rst:27]: Does not return anything.

[the remark this answers is not in the document]
Comment (augusthjerrild) [reply source/multicast/multicast.rst:27]: When PR is accepted, I will attach in the issue.

File: source/multicast/multicast.rst [hunk 3]
```diff
@@ -29,20 +29,20 @@ This sequence diagram describes the flow from the user to the Chirpstack Applica
...
-The downlink payload is send using Chirpstack which also has the multicast-group with the devices from OS2IoT.  
+The downlink payload is sent using Chirpstack which also has the multicast-group with the devices from OS2IoT.  
 
 Explanation of properties
 --------------------------
 
 A very short explanation of some of the properties that needs to be fulfilled when creating a multicast-group:
 
-**Multicast address:** A address that defines the multicast-group. Has to be a 8-char hex value.
+**Multicast address:** An address that defines the multicast-group. Has to be a 8-character hex value.
```
Comment (AramAlsabti) [inline source/multicast/multicast.rst:41]: "(...) **an** 8-character (...)"

File: source/multicast/multicast.rst [hunk 4]
```diff
@@ -29,20 +29,20 @@ This sequence diagram describes the flow from the user to the Chirpstack Applica
 Sending downlink payload
 -------------------------
 
-When the multicast-group is created it's possible to send a downlink payload to the group of devices. This is done by navigating to the details about the multicast-group, and from here it's possible to send choose a port and a payload that you wish to send to the devices.
+When the multicast-group is created it's possible to send a downlink payload to the group of devices. This is done by navigating to the details-page of the multicast-group, and from here you have to choose a specific port and a payload that you wish to send to the devices in the Multicast-group.
```
Comment (AramAlsabti) [inline source/multicast/multicast.rst:32]: Details page is two words and without a hyphen

**Review by AramAlsabti — APPROVED** [review]
Minor feedback. Looks good to me

**Pull request merged**